lxqt-config/lxqt-config-appearance/man/lxqt-config-appearance.1
.TH lxqt-config-appearance "1" "September 2012" "LXQt\ 0.7.0" "LXQt\ GUI
settings"
.SH NAME
lxqt-config-appearance \- GUI appearance application of \fBLXQt\fR: the fast
and lightweight Qt Desktop Environment
.SH SYNOPSIS
.B lxqt-config-appearance
.br
.SH DESCRIPTION
With this application you can set and configure the Qt/LXQt GUI appearance of
programs.
.P
In Linux, the look of a GUI application depends on the toolkit it is based on.
in LXQt, the overall look of many apps relies on the Qt framework GUI. The
\fBlxqt-config-appearance\fR application provides all aspects of configuring the
appearance of the \fBLXQt\fR desktop.
.P
All the remaining major aspects of LXQt appearance rely on the active Qt widget
style because the \fBLXQt\fR DE is based on the Qt framework.
.P
\fBLXQt\fR is an advanced, easy-to-use, and fast desktop environment based on Qt
technologies and ships several core desktop components, all of which are
optional:
.P
 * Panel
 * Desktop
 * Application launcher
 * Settings center \fI(related to this)\fR
 * Session handler
 * Polkit handler
 * SSH password manager
 * Display manager handler
 * Power manager
.P
These components perform actions like their counterparts in other desktop
environments and their names are self-descriptive. They are usually not
launched manually but automatically, when an \fBLXQt\fR session is chosen in the
Display Manager.
.P
.SH "REPORTING BUGS"
Report bugs to https://github.com/lxqt/lxqt/issues
.SH "SEE ALSO"
\fBLXQt\fR it has been tailored for users who value simplicity, speed, and
an intuitive interface. LXQt is also intended for less powerful machines. Also,
See:
.\" any module must refer to session app, for more info on starting it
.P
\fBlxqt-session.1\fR  LXQt module for managing the LXQt environment.
.P
\fBlxqt-config.1\fR  LXQt application for general config and settings.
.P
.SH AUTHOR
This manual page was created by \fBPICCORO Lenz McKAY\fR \fI<mckaygerhard@gmail.com>\fR
for \fBLXQt\fR project and VENENUX GNU/Linux but can be used by others.


CHANGELOG

lxqt-build-tools-0.6.0 / 2018-01-25
===================================

  * Bump version to 0.6.0
  * Don't use automatic string conversions
  * Introduced lxqt-transupdate for translation updates
  * Improved cmake scripting
    - Set cmake_minimum_required to 3.1.0
    - Consider QT_SELECT - some distributions use this tool
    - Drop 0X and 11 support, require c++14
    - Remvoed QUIET from any finds
    - Cleaned up LXQtTranslateTs module
    - Removed translation pulling and cleanup external repos
      after translations moved back to their original repos
    - Remove link-time optimization for gcc since it breaks
      qobject_cast() in some cases.
    - Removed obsolete FindFm module

lxqt-build-tools-0.5.0 / 2018-05-21
===================================

  * Bump version to 0.5.0
  * Prevent in-source builds
  * Adds an PreventInSourceBuilds module
  * http --> https
  * Fixed some LXDE mentions
  * LXQtCompilerSettings: Set CMAKE_C_VISIBILITY_PRESET to hidden
  * Don't allow Qt foreach/Q_FOREACH
  * Update CMakeLists.txt
  * Require Qt 5.7.1
  * Adds a FindXKBCommon CMake module
  * Adds ECMFindModuleHelpers CMake module
  * Don't export github templates

0.4.0 / 2017-09-22
==================

  * Release 0.4.0: Update changelog
  * Bump version to 0.4.0
  * Added needed versions
  * remove version set from LXQtConfigVars.in
  * Move LXQt version to liblxqt
  * Fixes a FTBFS with superbuild mode
  * Adds a FindExif module
  * LXQtCompilerSettings: Fixes a typo in diagnostics-color
  * Adds (Ninja) colored diagnostics for GCC (>=4.9)
  * Enable strict iterators for debug builds

0.3.2 / 2017-01-14
==================

  * Release 0.3.2: Update changelog
  * Bump version to 0.3.2 - finally the build tools are arch: all (#15)
  * Warnings (#16)
  * Remove arched bits from version.cmake-file (#14)

0.3.1 / 2017-01-01
==================

  * Release 0.3.1: Update changelog
  * Bump LXQT_PATCH_VERSION and build tools version (#13)
  * 32 64 bit ness (#12)
  * Use "gcc-ar" and "gcc-ranlib" tools when building static libs with gcc link-time optimization. (#11)

0.3.0 / 2016-12-06
==================

  * Release 0.3.0: Update changelog
  * Adds an FindXCB CMake module
  * FindUDev: Add VERSION_STRING support
  * FindUDev: Set REQUIRED_VARS
  * FindUdev: Feed pkg_modules_modules results to find_library/path
  * FindUDev: Adds Feature Summary
  * Adds FindUDev module
  * Enables colored diagnostics for CLang/Ninja combination
  * Disable link-time optimization for clang++/llvm since after some testing it breaks QObject signal/slot sometimes. * Remove trailing spaces.
  * Use different link-time optimization compiler options for gcc and clang and move -Bsymbolic flags to linker flags.
  * Add more linker options to turn on linker-time optimizations for gcc & clang.
  * Set CMAKE_BUILD_TYPE to Release by default if it's value is not set.
  * FindGLIB: Improve gio-unix handling
  * Adds a FindGLIB CMake module
  * Adds a FindMenuCache CMake module
  * Adds a FindFm CMake module
  * move project name to top
  * Fixes the name of package config file name in superbuild mode

0.2.0 / 2016-10-30
==================

  * Release 0.2.0: Update changelog
  * Bump minor to 2 (#7)
  * Added Qt5Core to README.md (#6)
  * Adds an LXQtConfigVars module
  * Adds superbuild support
  * README.md: Update dependencies

0.1.0 / 2016-10-14
==================

  * Release 0.1.0: Update changelog
  * Adds LXQtCreatePortableHeaders module
  * Authors and license (#2)
  * Add README.md
  * Make lxqt-build-tools be usable

0.0.0 / 2016-09-30
==================

  * Remove cpack (#107)
  * Bump version ot 0.11.0 (#105)
  * Settings: Add "homemade" XDG_CONFIG_DIRS support
  * translations: Change default translations repository
  * Don't lock the screen when it's already locked.
  * LXQtTranslateTs.cmake: Enable PULL_TRANSLATIONS by default
  * LXQtTranslateTs.cmake: Use/update template in remote repo
  * .gitignore: Cleanup
  * build: Use external translations
  * translations: Fix external translations logic
  * translations: Add external translations support
  * Adds CMAKE_INSTALL_PREFIX to the LXQt Theme search paths
  * LXQtCompilerSettings: Add QT_USE_QSTRINGBUILDER to the compile definitions
  * TranslateDesktopTs: Adds UPDATE_OPTIONS options
  * TranslateDesktop: Remove lupdate absolute option
  * TranslateDesktop: Use -a switch in grep commands
  * Use CMAKE_INSTALL_FULL_DATAROOTDIR
  * Add LXQT_GRAPHICS_DIR directory definition
  * Adds FindXdgUserDirs CMake find_package module
  * Release 0.10.0
  * pluginTranslation: removed using QStringLiteral as it was causing SEGFAULT in static finalization time
  * pluginTranslation: add some symbol for referencing (for file to be not stripped out in static linking time)
  * Adds "automatic" plugin translations loader support
  * Rename LxQt to LXQt everywhere
  * Handles CMake policy CMP0063
  * Adds an lxqt_enable_target_exceptions() CMake function
  * Adds LXQtCompilerSettings CMake module
  * LXQtCreatePkgConfigFile.cmake: Makes install COMPONENT configurable
  * LXQtTranslateTs.cmake Adds install COMPONENT option
  * Adds Runtime install COMPONENT
  * Fixes an SuperBuild/NormalBuild FTBFS
  * CMake: Use CMAKE_AUTOUIC, drop qt5_wrap()
  * CMake: Quote variables
  * CMake: Removes unused variables
  * Removes LIB_INSTALL_DIR CACHE PATH variable
  * Use relative paths in install DESTINATION
  * Marks link libraries as PUBLIC
  * Use target_compile_definitions() instead of add_definitions()
  * Make Release the default build type
  * Simplifies build type matching
  * lxqt-config.cmake.in: Use find_dependency()
  * Updates the build system to the Targets infrastructure
  * Sort .desktop translations. Fix typos in the comments. Remove local dir artifact.
  * pkg-config: Adds Qt5Xdg to the required dependencies
  * Updates lxqt_create_pkgconfig_file()
  * Stops using Qt5Xdg dropped stuff
  * Adds LXQT_DATA_DIR configuration variable

lxqt-config/lxqt-config-appearance/translations/lxqt-config-appearance.ts
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en_US">
<context>
    <name>ConfigOtherToolKits</name>
    <message>
        <location filename="../configothertoolkits.cpp" line="147"/>
        <source>GTK themes</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../configothertoolkits.cpp" line="148"/>
        <source>&lt;p&gt;&apos;%1&apos; has been overwritten.&lt;/p&gt;&lt;p&gt;You can find a copy of your old settings in &apos;%2&apos;&lt;/p&gt;</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>FontsConfig</name>
    <message>
        <location filename="../fontsconfig.ui" line="23"/>
        <source>Font</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="30"/>
        <source>Default font for user interface</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="36"/>
        <source>Font name:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="46"/>
        <source>Style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="53"/>
        <source>Point size:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="68"/>
        <source>Normal</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="73"/>
        <source>Bold</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="78"/>
        <source>Italic</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="83"/>
        <source>Bold Italic</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="94"/>
        <source>The following settings only affect newly started applications</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="100"/>
        <source>Use antialias fonts</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="107"/>
        <source>Font hinting style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="118"/>
        <location filename="../fontsconfig.ui" line="183"/>
        <source>None</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="123"/>
        <source>Slight</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="128"/>
        <source>Medium</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="133"/>
        <source>Full</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="141"/>
        <source>Font hinting</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="148"/>
        <source>Resolution (DPI):</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="155"/>
        <source>Autohint</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="172"/>
        <source>Subpixel antialiasing:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="188"/>
        <source>RGB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="193"/>
        <source>BGR</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="198"/>
        <source>VRGB</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../fontsconfig.ui" line="203"/>
        <source>VBGR</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>IconThemeConfig</name>
    <message>
        <location filename="../iconthemeconfig.ui" line="14"/>
        <source>LXQt Appearance Configuration</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="26"/>
        <source>Icons Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="87"/>
        <source>The KDE extension of XDG icon themes -&gt; FollowsColorScheme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../iconthemeconfig.ui" line="90"/>
        <source>Colorize icons based on widget style (palette)</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>LXQtThemeConfig</name>
    <message>
        <location filename="../lxqtthemeconfig.ui" line="23"/>
        <source>LXQt Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../lxqtthemeconfig.ui" line="57"/>
        <source>Override user-defined wallpaper</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>QObject</name>
    <message>
        <location filename="../main.cpp" line="58"/>
        <source>LXQt Appearance Configuration</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="71"/>
        <source>Widget Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="79"/>
        <source>Icons Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="88"/>
        <source>LXQt Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="96"/>
        <source>Font</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../main.cpp" line="106"/>
        <source>Cursor</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>StyleConfig</name>
    <message>
        <location filename="../styleconfig.ui" line="23"/>
        <source>Widget Style</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="42"/>
        <source>Set GTK themes (GTK configuration files will be overwritten!)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="62"/>
        <source>GTK 3 Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="72"/>
        <source>GTK 2 Theme</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="82"/>
        <source>To attempt uniform theming, either select similar style/theme (if available) across all lists, or select &apos;gtk2&apos; Qt style (if available) to mimic GTK themes.

Make sure &apos;xsettingsd&apos; is installed to help GTK applications apply themes on the fly.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="99"/>
        <source>Toolbar button style:</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="107"/>
        <source>Only display the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="112"/>
        <source>Only display the text</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="117"/>
        <source>The text appears beside the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="122"/>
        <source>The text appears under the icon</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="127"/>
        <source>Default</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="135"/>
        <source>Activate item on single click</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../styleconfig.ui" line="145"/>
        <source>Qt Style</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
